modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -1,41 +1,33 @@
 import { SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, Bucket, NumericAggregations } from '../types/aggregations';
 import { AllAggregations } from '../types/types';
-import { RequestedFieldsMap } from '../util';
 
 type ResolveAggregationInput = {
 	aggregationsMap: AllAggregations;
-	requestedAggregationFields: string[];
 	accumulator: AllAggregations;
 };
 
+type AggregationsTuple = [Aggregations, Aggregations];
+type NumericAggregationsTuple = [NumericAggregations, NumericAggregations];
+
 /**
  * Resolves returned aggregations from network queries into single accumulated aggregation
  *
  * @param
  */
-const resolveAggregations = ({
-	aggregationsMap,
-	requestedAggregationFields,
```

Comment:
No more requested fields, just aggregate what's there -- definitely simplifies things

I'm just curious why there was a need to 'request specific fields' before and how that is handled now

Reply:
There is still a need to request specific fields, it just happens before the aggregation resolving.
POST GQL Query to request fields => resolve response

Example:
query to central server: `{donors_gender: {...}, donors_age: {...}}`
NodeA has `[donors_gender, donors_age]`
NodeB only has `[donors_age]`

Before code change if we are iterating on original query `NodeB.response[donors_gender]` will be `undefined`

Reply:
Got it so it's all just stream lined handling of the same scenario, just wanted to make sure

Reply:
Yes. Other relevant comment for reference too https://github.com/overture-stack/arranger/pull/898#discussion_r1786070816 

For sure - thanks for checking!